isamples_metadata/SESARTransformer.py

```diff
@@ -293,7 +294,9 @@ def keywords(self) -> typing.List:
         return [self._source_record_description()["sampleType"]]
 
     def produced_by_id_string(self) -> str:
-        # TODO: this is present for GEOME, does anything make sense for SESAR?
```

Comment:
In answer to the comment above -- yes, `parentIdentifier` makes sense for SESAR.  :)